internal/manifests/internal/sizes.go

```diff
@@ -195,6 +224,9 @@ var StackSizeTable = map[lokiv1beta1.LokiStackSizeType]lokiv1beta1.LokiStackSpec
 			Gateway: &lokiv1beta1.LokiComponentSpec{
 				Replicas: 2,
 			},
+			IndexGateway: &lokiv1beta1.LokiComponentSpec{
+				Replicas: 1,
+			},
```

Comment:
Can you elaborate if we need to pin the number of replicas to 1 for the Index Gateway like we do for Compactor when we apply the default options?

Reply:
There isn't a need to keep the Index Gateway to one replica like we do for the Compactor.

Reply:
Then why not having 2 replicas for HA in `1x.small` and `1x.medium`?